proto/Makefile

```diff
@@ -16,35 +16,22 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PROTO_LIB   := lib/libroachproto.a
-PROTOS      := api.proto config.proto data.proto errors.proto gossip.proto heartbeat.proto internal.proto
+PROTOC      := $(shell git rev-parse --show-cdup)_vendor/usr/bin/protoc
```

Comment:
The other .pb.go files are already checked in (e.g. rpc/code/wire.pb/wire.pb.go). I'll switch back to using `..` instead of `git rev-parse`.